`ks diff` segfaults

Running `ks diff` with no params segfaults.

$ ks version
kubecfg version: dev-2017-11-22T13:35:25-0800
jsonnet version: v0.9.5
client-go version: v1.6.8-beta.0+$Format:%h$

$  ks diff
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1b3ffcb]

goroutine 1 [running]:
github.com/ksonnet/ksonnet/cmd.initDiffCmd(0x255c4a0, 0xc420012044, 0x2e, 0x0, 0x0, 0x258cdb8, 0x0, 0x0, 0x1dfc420, 0x3, ...)
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/cmd/diff.go:129 +0x6db
github.com/ksonnet/ksonnet/cmd.glob..func3(0x255c4a0, 0x258cdb8, 0x0, 0x0, 0x0, 0x0)
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/cmd/diff.go:79 +0x1dc
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).execute(0x255c4a0, 0x258cdb8, 0x0, 0x0, 0x255c4a0, 0x258cdb8)
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:656 +0x3f1
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x255f9c0, 0xc4205a7f10, 0x1, 0x1)
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:735 +0x2fe
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).Execute(0x255f9c0, 0x0, 0x5b)
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:694 +0x2b
main.main()
        /Users/fabio/go/src/github.com/ksonnet/ksonnet/main.go:33 +0x5b

Expected: help text (and no segfault)

--- a/cmd/diff.go
+++ b/cmd/diff.go
@@ -44,8 +44,11 @@ var diffCmd = &cobra.Command{
 	Use:   "diff [<env1> [<env2>]] [-f <file-or-dir>]",
 	Short: "Display differences between server and local config, or server and server config",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("'diff' requires at least one argument, that is the name of the environment\n\n%s", cmd.UsageString())
+		}
 		if len(args) > 2 {
-			return fmt.Errorf("'diff' takes at most two arguments, that are the name of the environments")
+			return fmt.Errorf("'diff' takes at most two arguments, that are the name of the environments\n\n%s", cmd.UsageString())
 		}
 
 		flags := cmd.Flags()

